### PR TITLE
✨ Pixi: check for latest version of components

### LINKED ISSUE #5448: Pixi: show warning if not using latest version of a component
Some components had CLS issues which have been fixed in later versions. We can use the validator rules to get the latest version for each component.

### PR DESCRIPTION
**Pull request by lluerich**
* older component versions might cause known CLS issues
* fixes #5448

### CODE REVIEW

File: pixi/backend/api.js [hunk 1]
```diff
@@ -56,6 +57,28 @@ const isCacheUrl = (urlString, cachesList) => {
   return !!matchedCache;
 };
 
+const getComponentVersions = async () => {
```
Comment (sebastianbenz) [inline pixi/backend/api.js:60]: Can we move the check into linter?

**Comment by lluerich** [comment]
This can be reviewed once ampproject/amp-toolbox#1165 is in. I will update the PR accordingly in the meantime.

File: pixi/backend/api.js (hunk 1 above)
Comment (lluerich) [reply pixi/backend/api.js:60]: Moved here ampproject/amp-toolbox/pull/1165.

**Comment by sebastianbenz** [comment]
Just published a new linter release. If you rebase against master you can test the check.

**Comment by lluerich** [comment]
This is good to go, if you are fine with the provided warning text @sebastianbenz.

**Review by sebastianbenz — APPROVED** [review]
Looks great - thanks!

**Pull request merged**